tests/components/goodwe/__init__.py

```diff
@@ -1 +1,76 @@
 """Tests for the Goodwe integration."""
+from typing import Any
+
+from goodwe import Inverter, OperationMode, Sensor
+
+
+class MockInverter(Inverter):
```

Comment:
As you are returning only static data I think it would be better to use the [python mock](https://docs.python.org/3/library/unittest.mock.html#quick-guide) instead

Reply:
Done, using MagicMock now.